src/data/local/github/tracks/track-4/steps/step-2.md

```diff
@@ -1,96 +1,23 @@
-# Step 2 - Integrate the app with the server
+# Step 2 - GraphQL
+#### Estimated time: TODO: add estimated time
 
-Your task is to get all data from the inputs in the screen and send it to the server with an HTTP request.
+## Getting started
+As we got out feet wet with REST, let's get started with GraphQL.
```

Comment:
Ficou meio solto isto daqui... como do REST ele chega no GraphQL? A intenção é deixar isso no ar mesmo?

Reply:
@tibawatanabe no step anterior tem uma 'explicação' do porque ele está aprendendo REST. 
> Network communication is a crucial part of any system nowadays. Here in Taqtile we use a framework called GraphQL, which you'll use thoughout this onboard.
But for now, in order to you understand what's happening under the GraphQL's hood, let's start with the (not so) old fashion way of doing network requests: The Representational State Transfer a.k.a. **REST**.

Você acha que está bom esse step de introdução?